test_scripts/Defects/4_6/964_SDL_sends_to_mobile_APPLICATION_NOT_REGISTRED_in_setAppIcon_responce_if_HMI_responds_with_INVALID_DATA.lua

```diff
@@ -0,0 +1,72 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/964
+--
+-- Precondition:
+-- SDL Core and HMI are started. App is registered, HMI level = FULL
+-- Description:
+-- Steps to reproduce:
+-- 1) Send PutFile with file name =\syncFileName
+--    SDL returns SUCCESS result, success:true
+-- 2) Send SetAppIcon with file name =\syncFileName
+-- 3) Send from HMI INVALID_DATA to SDL.
+-- Expected:
+-- 1) SDL should send to mobile result code which was sent to SDL by HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Defects/commonDefects')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local utils = require("user_modules/utils")
+local actions = require("user_modules/sequences/actions")
+
+--[[ Local Functions ]]
+local function getPathToFileInStorage(pFileName)
+  return commonPreconditions:GetPathToSDL() .. "storage/"
+    .. actions.getConfigAppParams().appID .. "_"
+    .. utils.getDeviceMAC() .. "/" .. pFileName
+end
+
+local function putFile(self)
+  local paramsSend = {
+    syncFileName = "\\syncFileName",
```

Comment:
@HSavynetska Do we really need new ```syncFileName``` in this script?
I suppose it will work with one of the existing files

Reply:
@HSavynetska  Please check if issue is reproduced with existing file